custom/rch/models.py

```diff
@@ -0,0 +1,207 @@
+            if self.Delivery_Time:
+                datetime.datetime.strptime(self.Delivery_Time, '%H:%M:%S')
+        except ValueError:
+            self.Delivery_Time = None
+
+        try:
+            if self.Discharge_Time:
+                datetime.datetime.strptime(self.Discharge_Time, '%H:%M:%S')
+        except ValueError:
+            self.Discharge_Time = None
+
+        if self.Maternal_Death:
+            self.Maternal_Death = 0
+            if self.Maternal_Death == 'Y' or self.Maternal_Death == 'y':
+                self.Maternal_Death = 1
+
+        super(RCHMother, self).save(*args, **kwargs)
+
+    @classmethod
+    def update_beneficiaries(cls, beneficiary_type=MOTHER_DATA_TYPE):
+        super(RCHMother, cls).update_beneficiaries(beneficiary_type)
+
+
+class RCHChild(RCHRecord):
+    ANM_Name = models.CharField(null=True, max_length=255)
+    ANM_Phone_No = models.BigIntegerField(null=True)
+    ASHA_Name = models.CharField(null=True, max_length=255)
+    ASHA_Phone_No = models.BigIntegerField(null=True)
+    Birth_Date = models.DateTimeField(null=True)
+    Child_Aadhaar_No = models.BigIntegerField(null=True)
+    Child_RCH_ID_No = models.BigIntegerField(null=True)
+    DPTBooster1_Antibiotics_Given = models.CharField(null=True, max_length=255)
+    DPTBooster1_Breastfeeding = models.CharField(null=True, max_length=255)
+    DPTBooster1_Complentary_Feeding = models.CharField(null=True, max_length=255)
+    DPTBooster1_Diarrhoea = models.CharField(null=True, max_length=255)
+    DPTBooster1_ORS_Given = models.CharField(null=True, max_length=255)
+    DPTBooster1_Pneumonia = models.CharField(null=True, max_length=255)
+    Delete_mother = models.CharField(null=True, max_length=255)
+    Fully_Immunized = models.CharField(null=True, max_length=255)
+    Gender = models.CharField(null=True, max_length=255)
+    Measles1_Antibiotics_Given = models.CharField(null=True, max_length=255)
+    Measles1_Breastfeeding = models.CharField(null=True, max_length=255)
+    Measles1_Complentary_Feeding = models.CharField(null=True, max_length=255)
+    Measles1_Diarrhoea = models.CharField(null=True, max_length=255)
+    Measles1_ORS_Given = models.CharField(null=True, max_length=255)
+    Measles1_Pneumonia = models.CharField(null=True, max_length=255)
+    Mobile_Relates_To = models.CharField(null=True, max_length=255)
+    Mother_Case_no = models.SmallIntegerField(null=True)
+    Mother_RCH_ID_NO = models.BigIntegerField(null=True)
+    Name_Child = models.CharField(null=True, max_length=255)
+    Name_Father = models.CharField(null=True, max_length=255)
+    Name_Mother = models.CharField(null=True, max_length=255)
+    Received_AllVaccines = models.CharField(null=True, max_length=255)
+    SubCentre_ID = models.SmallIntegerField(null=True)
+    VILLAGE_Name = models.CharField(null=True, max_length=255)
+    Weight = models.FloatField(null=True)
+
+    @classmethod
+    def update_beneficiaries(cls, beneficiary_type=CHILD_DATA_TYPE):
```

Comment:
probably don't need the `beneficiary_type` kwarg since I doubt you'll ever call it with something different.